portal-impl/src/com/liferay/portal/service/impl/CountryServiceImpl.java

```diff
@@ -123,7 +123,6 @@ public Country fetchCountryByA3(String a3) {
 		return countryLocalService.getCompanyCountries(companyId);
 	}
 
-	@AccessControlled(guestAccessEnabled = true)
```

Comment:
I think we wanted to keep this one here since it's replacing the non-companyId method. Or is there another non-deprecated method that guests can access for Countries?